package.json

```diff
@@ -43,47 +43,48 @@
     "bhanuhiteshi <bhanu.singh@hiteshi.com>",
     "James Hyde <plinkpaste@jameshy.de>",
     "ej2 <edward.emanuel@gmail.com>",
-    "Peter Blazejewicz <peterblazejewicz@users.noreply.github.com>"
+    "Peter Blazejewicz <peterblazejewicz@users.noreply.github.com>",
+    "Arjen Brandenburgh <mail@arjenbrandenburgh.nl>"
   ],
   "license": "MIT",
   "bugs": {
     "url": "https://github.com/FortAwesome/angular-fontawesome/issues"
   },
   "homepage": "https://github.com/FortAwesome/angular-fontawesome",
   "devDependencies": {
-    "@angular-devkit/build-angular": "~0.901.0",
-    "@angular-devkit/build-ng-packagr": "~0.901.0",
-    "@angular-devkit/core": "~9.1.0",
-    "@angular-devkit/schematics": "~9.1.0",
-    "@angular/cli": "~9.1.0",
-    "@angular/common": "~9.1.0",
-    "@angular/compiler": "~9.1.0",
-    "@angular/compiler-cli": "~9.1.0",
-    "@angular/core": "~9.1.0",
-    "@angular/language-service": "~9.1.0",
-    "@angular/platform-browser": "~9.1.0",
-    "@angular/platform-browser-dynamic": "~9.1.0",
+    "@angular-devkit/build-angular": "~0.1000.0",
+    "@angular-devkit/build-ng-packagr": "~0.1000.0",
+    "@angular-devkit/core": "~10.0.0",
+    "@angular-devkit/schematics": "~10.0.0",
+    "@angular/cli": "~10.0.0",
+    "@angular/common": "~10.0.0",
+    "@angular/compiler": "~10.0.0",
+    "@angular/compiler-cli": "~10.0.0",
+    "@angular/core": "~10.0.0",
+    "@angular/language-service": "~10.0.0",
+    "@angular/platform-browser": "~10.0.0",
+    "@angular/platform-browser-dynamic": "~10.0.0",
     "@fortawesome/fontawesome-svg-core": "^1.2.28",
     "@fortawesome/free-regular-svg-icons": "^5.13.0",
     "@fortawesome/free-solid-svg-icons": "^5.13.0",
     "@types/jasmine": "~3.5.10",
     "@types/node": "~13.9.5",
     "codelyzer": "^5.2.2",
-    "jasmine-core": "3.5.0",
-    "jasmine-spec-reporter": "~5.0.1",
-    "karma": "~4.4.1",
+    "jasmine-core": "~3.5.0",
+    "jasmine-spec-reporter": "~5.0.0",
+    "karma": "~5.0.0",
     "karma-chrome-launcher": "~3.1.0",
-    "karma-coverage-istanbul-reporter": "~2.1.1",
-    "karma-jasmine": "~3.1.1",
-    "karma-jasmine-html-reporter": "1.5.3",
-    "ng-packagr": "^9.1.0",
+    "karma-coverage-istanbul-reporter": "~3.0.2",
+    "karma-jasmine": "~3.3.0",
+    "karma-jasmine-html-reporter": "^1.5.0",
+    "ng-packagr": "^10.0.0",
     "prettier": "2.0.2",
-    "protractor": "~5.4.3",
+    "protractor": "~7.0.0",
     "rxjs": "^6.5.4",
     "ts-node": "~8.8.1",
-    "tslib": "^1.11.1",
+    "tslib": "^2.0.0",
```

Comment:
Please also move `tslib` to [dependencies](https://github.com/FortAwesome/angular-fontawesome/pull/258/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R53) as this is now the recommended way.